x/stakeibc/keeper/msg_server_register_host_zone.go

```diff
@@ -50,7 +50,7 @@ func (k msgServer) RegisterHostZone(goCtx context.Context, msg *types.MsgRegiste
 		HostDenom:         msg.HostDenom,
 		TransferChannelId: msg.TransferChannelId,
 		// Start exchange rate at 1 upon registration
-		RedemptionRate:     sdk.NewDec(1),
+		RedemptionRate:     sdk.NewDec(10),
```

Comment:
Should the base (success) case have RedemptionRate == 1?

Reply:
yes, shifted back to 1